### PR TITLE
[Page Shield] Update alerts

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-7338.

### CODE REVIEW

**Comment by pedrosousa** [comment]
Relevant preview pages:
https://pedro-pcx-7338-page-shield-u.cloudflare-docs-7ou.pages.dev/fundamentals/notifications/notification-available/#page-shield
https://pedro-pcx-7338-page-shield-u.cloudflare-docs-7ou.pages.dev/page-shield/detection/configure-alerts/

File: content/fundamentals/notifications/notification-available.md [hunk 1]
```diff
@@ -506,364 +506,364 @@ All Cloudflare plans.
...
 
 **Included with**
 
-Enterprise plans with paid add-on.
+When an advanced certificate is validated, issued, renewed, or expired.
 
 **What should you do if you receive one?**
 
-{{<render file="_script-monitor-review-malicious-scripts.md">}}
+Action only needed if notification is about a certificate that failed to be issued. Refer to [SSL expired or SSL mismatch errors](https://support.cloudflare.com/hc/articles/200170566#h_c1a6e78e-150d-4db6-89ab-eec7cb1ab03f) for more information.
```
Comment (marciocloudflare) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Action only needed if notification is about a certificate that failed to be issued. Refer to [SSL expired or SSL mismatch errors](/support/ssl-tls/troubleshooting/troubleshooting-ssl-errors/#ssl-expired-or-ssl-mismatch-errors) for more information.
```

Comment (marciocloudflare) [reply content/fundamentals/notifications/notification-available.md]: The link redirects here. should we replace it?

**Review by marciocloudflare — APPROVED** [review]

File: content/fundamentals/notifications/notification-available.md (hunk 1 above)
Comment (pedrosousa) [reply content/fundamentals/notifications/notification-available.md]: I'd say yes, even though I only changed the Page Shield part.

**Pull request merged**